pkg/v1/providers/ytt/vendir/kapp-controller/_ytt_lib/addons/packages/kapp-controller/0.30.0/bundle/config/schema.yaml

```diff
@@ -0,0 +1,44 @@
+#! schema.yaml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for kapp-controller"
+---
+#@schema/desc "The namespace in which to deploy kapp-controller"
+namespace: kapp-controller
+#@schema/desc "Configuration for kapp-controller"
+kappController:
+  #@schema/desc "The namespace value used by older templates, will overwrite with top level namespace if present, for backward compatibility"
+  #@schema/nullable
+  namespace: kapp-controller
+  #@schema/desc "Whether to create namespace specified for kapp-controller"
+  createNamespace: true
+  #@schema/desc "The namespace value used for global packaging resources. Any Package and PackageMetadata CRs within that namespace will be included in all other namespaces on the cluster, without duplicating them"
+  globalNamespace: tanzu-package-repo-global
+  deployment:
+    #@schema/desc "The coreDNSIP will be injected into /etc/resolv.conf of kapp-controller pod"
+    #@schema/nullable
+    coreDNSIP: ""
+    #@schema/desc "Whether to enable host networking for kapp-controller deployment"
+    hostNetwork: false
+    #@schema/desc "The priority value that various system components use to find the priority of the kapp-controller pod"
+    #@schema/nullable
+    priorityClassName: ""
+    #@schema/desc "Concurrency of kapp-controller deployment"
+    concurrency: 4
+    #@schema/desc "kapp-controller deployment tolerations"
+    tolerations: ["toleration1"]
```

Comment:
suggestion: toleration seems not an array of string, not sure how to define an array of struct/map in schema.

```
      tolerations:
      - key: CriticalAddonsOnly
        operator: Exists
      - effect: NoSchedule
        key: node-role.kubernetes.io/master
      - effect: NoSchedule
        key: node.kubernetes.io/not-ready
      - effect: NoSchedule
        key: node.cloudprovider.kubernetes.io/uninitialized
        value: "true"
```

Reply:
I think we should be able to use annotation:
`#@schema/type any=True` as per
https://carvel.dev/ytt/docs/latest/how-to-write-schema/#allowing-multiple-types-of-maps-or-arrays

I will add above for `tolerations` key